proms-resources/frontend/src/main/frontend/src/proms/QuestionnaireSet.jsx

```diff
@@ -470,11 +468,8 @@ function QuestionnaireSet(props) {
       : null
   }
 
-  const diffString = (startDate, endDate, division, result, modulus = null) => {
-    let diff = endDate.diff(startDate, division);
-    if (modulus != null) {
-      diff = diff % modulus;
-    }
+  const diffString = (division, result, diffs) => {
+    let diff = diffs[division];
```

Comment:
This is looking a little bit too precise:

![image](https://user-images.githubusercontent.com/651980/165782703-39587c55-8c92-4d1e-9635-7d47050ea549.png)
```suggestion
    let diff = Math.floor(diffs[division]);
```

Reply:
Fixed in da39f3c.